bootstrap/integration/bootstrap_tester.go

```diff
@@ -44,7 +44,7 @@ type BootstrapTester struct {
 }
 
 func NewBootstrapTester() (*BootstrapTester, error) {
-	homeDir, err := os.MkdirTemp("", "home")
+	homeDir, err := os.MkdirTemp("/tmp", "home")
```

Comment:
Maybe worth a comment mentioning the Linux-path-length-for-Unix-sockets problem?